bin/groopm2

```diff
@@ -179,6 +180,7 @@ class parse_command_configure:
         success = dm.createDB(
             timer,
             options.bamfiles,
+            options.cov_file,
```

Comment:
I suppose the user should give either a list of bam files or a single coverage file? Can ArgParser.add_mutually_exclusive_group() handle this?

Reply:
Indeed. Do you want to make the bamfiles option a named argument?

Reply:
I'd rather have the bamfiles option positional, if that's ok with argparse. Keep the change backwards compatible.

Reply:
Just tried this, but add_mutually_exclusive_group doesn't support optional positional arguments.

Reply:
OK. Maybe we add a manual check then, and I will merge.